tests/utils/testutils.py

```diff
@@ -219,54 +190,62 @@ def get_script_dir(cls):
         raise NotImplementedError
 
     @classmethod
-    def get_libraries_to_install(cls):
+    def get_libraries_to_install(cls) -> typing.List:
         pass
 
     @classmethod
     def get_environment_variables(cls):
         pass
 
     @classmethod
-    def get_script_name(cls):
-        pass
+    def docker_tests_enabled(self) -> (bool, str):
+        if is_envvar_true(CONSUMPTION_DOCKER_TEST):
+            return True, CONSUMPTION_DOCKER_TEST
+        elif is_envvar_true(DEDICATED_DOCKER_TEST):
+            return True, DEDICATED_DOCKER_TEST
+        else:
+            return False, None
 
     @classmethod
     def setUpClass(cls):
         script_dir = pathlib.Path(cls.get_script_dir())
+        is_unit_test = True if 'unittests' in script_dir.parts else False
```

Comment:
whats this .parts checking for? inline comment?

Reply:
It checks for different paths and if any of the paths has unittests